artifacttypes/tosca.artifacts/installPipeline/installPipeline.tosca

```diff
@@ -0,0 +1,8 @@
+tosca_definitions_version: tosca_simple_yaml_1_3
+
+artifact_types:
+  installPipeline:
```

Comment:
Please use `PascalCase` for type names.

Reply:
Fixed this for installPipeline. Did not notice the same issue elsewhere.